src/App.css

```diff
@@ -1,10 +1,136 @@
+/* OPEN GAMES LIST CARD */
 
-.game {
-    border: 2px green solid;
-    margin: 5px;
+.game-card {
+    border: 2px rgb(58, 151, 116) solid;
+    /* margin: 5px; */
+    width: 400px;
+    height: 110px;
+    background-color:#2C7A7B;
+    box-shadow: 0 8px 16px -8px rgba(0,0,0,0.4);
+    border-radius: 6px;
+    /* overflow: hidden; */
+    position: relative;
+    margin: 1rem;
+    /* display: grid; */
+    color: #fff;
+    padding-left: 10px;
+    padding-top: 5px;
+}
+
+button-showmore{
+    width: 80px;
+    height: 20px;
+    margin: 20px;
+    /* background: #c8bb04; */
+    background: #4FD1C5;
+    color: white;
+    font-size: 10px;
+    font-weight: bold;
+    text-align: center;
+    border-radius: 0.625em;
+    position: absolute;
+    left: 73%;
+    bottom: 59%;
+}
+
+button-join {
```

Comment:
We'll wanna work on consolidating these button styles some to reduce file bloat, tho not an urgent issue

Reply:
OK, you might want to show me about later.